examples/darwin-framework-tool/templates/tests/partials/test_cluster.zapt

```diff
@@ -23,10 +23,16 @@ class {{filename}}: public TestCommandBridge
     {
     }
 
+    // Allow yaml to access the current commissioner node id.
```

Comment:
nit, might be nice to add in comment that getting this value is done before every TestStep because you don't know when this value may have changed

Reply:
added